test/lib/import/colorectal/providers/royal_marsden/royal_marsden_handler_colorectal_test.rb

```diff
@@ -55,7 +55,7 @@ def setup
 
   test 'process_test_scope' do
     @handler.process_test_scope(@genotype, @record)
-    assert_equal 'Targeted Colorectal mutation test', @genotype.attribute_map['genetictestscope']
+    assert_equal 'Targeted Colorectal Lynch or MMR', @genotype.attribute_map['genetictestscope']
```

Comment:
Hi @FrancescoNea - this doesn't look related to the Oxford colorectal handler?

Reply:
It is part of a broader change of labels I had to perform. In Oxford, Ashkenazi Jews and Polish ethnicities are tested as well, so had to re-label AJ and Polish screens accordingly to the Colorectal tests, in order to differentiate them from the BRCA1/2 tests

Reply:
Thanks @FrancescoNea